src/components/DropdawnMenu/DropdawnMenu.tsx

```diff
@@ -0,0 +1,26 @@
+import { memo} from 'react';
+import { Person } from '../../types/Person';
+
+interface Props {
+  people: Person[];
+  onSelected: (person: Person) => void;
+}
+
+export const DropdownMenu: React.FC<Props> = memo(({ people, onSelected }) => (
+    <div className="dropdown-menu" role="menu" data-cy="suggestions-list">
+      <div className="dropdown-content">
+        {people.map(person => (
+          <div
+            key={person.name}
```

Comment:
Using `person.name` as the key might not be unique if there are people with the same name. Consider using a unique identifier like `person.id` if available, or ensure that `person.name` is unique.